tests/appsec/iast/test_grpc_iast.py

```diff
@@ -72,3 +73,21 @@ def test_taint_iast_last(self):
                     res = stub1.SayHelloLast(requests_iterator)
                     assert hasattr(res, "message")
                     _check_test_range(res.message)
+
+    def test_taint_iast_patching_import_error(self):
+        with mock.patch.dict("sys.modules", {"google._upb._message": None}), override_env({"DD_IAST_ENABLED": "True"}):
```

Comment:
do we know the conditions this is actually not available? certain versions of protobuf? when using pure-python version vs not? when a certain env var is set or something?

I'd almost rather see us have a riot/hatch matrix item to make this be true for all of our tests so we can verify there aren't any other code paths which assume this is available.

wdyt? I am happy to consider this as a follow-up, but would love to see an investigation into it.

Reply:
Yeah, I expected to be able to reproduce in a higher level test, but I couldn't and don't know the conditions where it's not available, so I'd at least have a simple regression test and investigate further once we unlocked the release

Reply:
I remember digging through the code, and I cannot remember the scenarios when it isn't available.

it might just be older versions don't have it? maybe we aren't testing with old enough versions of protobuf?

Reply:
again, as a follow-up could be good to look into, and maybe document the outcome (even if we cannot get it to work) at least

Reply:
Yeah, we definitely should investigate what is going on. Let's do a follow-up